### PR TITLE
feature/loan-purpose

### PR DESCRIPTION
**Pull request by gobravedave**
add translation of loan purpose as per #242

### CODE REVIEW

File: src/components/data/banking/LendingRate.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const LendingRate = (props) => {
       {!!applicationFrequency && <div>Applied <Duration prefix="every" value={applicationFrequency}/></div>}
       {!!interestPaymentDue && <div>Interest Payment {translateInterestPaymentDue(interestPaymentDue)}</div>}
       {!!repaymentType && <div>Repayment Type {translateRepaymentType(repaymentType)}</div>}
-      {!!loanPurpose && <div>Loan Purpose {loanPurpose}</div>}
+      {!!loanPurpose && <div>Loan Purpose {translateloanPurpose (loanPurpose)}</div>}
```
Comment (vadkor) [inline src/components/data/banking/LendingRate.js:64]: Sorry for neat-picking, but could you please remove the space between  translateloanPurpose and the opening bracket?

File: src/utils/dict.js [hunk 2]
```diff
@@ -173,6 +178,8 @@ export const translateInterestPaymentDue = (interestPaymentDue) => interestPayme
 
 export const translateRepaymentType = (repaymentType) => repaymentTypeDict[repaymentType]
 
+export const translateloanPurpose = (loanPurpose) => loanPurposeDict [loanPurpose]
```
Comment (vadkor) [inline src/utils/dict.js:181]: And this unneeded space before the square bracket...

**Review by vadkor — CHANGES_REQUESTED** [review]
Thank you, good addition. Just a couple of cosmetic things.

File: src/components/data/banking/LendingRate.js (hunk 1 above)
Comment (gobravedave) [reply src/components/data/banking/LendingRate.js]: updated

File: src/utils/dict.js (hunk 2 above)
Comment (gobravedave) [reply src/utils/dict.js]: updates

**Review by vadkor — APPROVED** [review]
Great! Thank you.

**Pull request merged**